src/GovUk.Education.ExploreEducationStatistics.Analytics.Consumer/ProcessorHostBuilder.cs

```diff
@@ -16,11 +16,12 @@ public static class ProcessorHostBuilder
     public static IHostBuilder ConfigureProcessorHostBuilder(this IHostBuilder hostBuilder)
     {
         return hostBuilder
-            .ConfigureAppConfiguration(builder =>
+            .ConfigureAppConfiguration((context, builder) =>
             {
                 builder
                     .AddJsonFile("appsettings.json", optional: true, reloadOnChange: false)
                     .AddJsonFile("appsettings.Local.json", optional: true, reloadOnChange: false)
+                    .AddJsonFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.json", optional: true)
```

Comment:
I would put this line *above* `appsettings.Local.json`, as we always allow `appsettings.Local.json` to have the final say!

Reply:
Thanks!